WordPress/Classes/System/WordPressAppDelegate+openURL.swift

```diff
@@ -0,0 +1,162 @@
+import WordPressAuthenticator
+
+/// Custom pattern matcher used for our URL scheme
+private func ~=(pattern: String, value: URL) -> Bool {
+    return value.absoluteString.contains(pattern)
+}
+
+@objc extension WordPressAppDelegate {
+    internal func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey : Any] = [:]) -> Bool {
+        DDLogInfo("Application launched with URL: \(url)");
+        var returnValue = false
+
+        // 1. check if hockey can open this URL
+        var hockeyOptions: [String: Any] = [:]
+        for (key, value) in options {
+            hockeyOptions[key.rawValue] = value
+        }
+        if hockey.handleOpen(url, options: hockeyOptions) {
+            returnValue = true
+        }
+
+        // 2. check if this is a Google login URL
+        if WordPressAuthenticator.shared.handleGoogleAuthUrl(url,
+                                                             sourceApplication: options[UIApplication.OpenURLOptionsKey.sourceApplication] as? String,
+                                                             annotation: options[UIApplication.OpenURLOptionsKey.annotation]) {
+            returnValue = true
+        }
+
+        // 3. let's see if it's our wpcom scheme
+        guard url.absoluteString.hasPrefix(WPComScheme) else {
```

Comment:
```suggestion
        guard url.scheme == WPComScheme else {
```

Reply:
I wasn't sure if this would break something or not. Definitely seems like the right thing to do tho 😜